src/chrome/content/rules/OptOutPrescreen.com.xml

```diff
@@ -1,15 +1,18 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://optoutprescreen.com/ => https://www.optoutprescreen.com/: (28, 'Operation timed out after 0 milliseconds with 0 out of 0 bytes received')
-Fetch error: http://www.optoutprescreen.com/ => https://www.optoutprescreen.com/: (28, 'Operation timed out after 0 milliseconds with 0 out of 0 bytes received')
+	Note that the OptOutPrescreen.com site appears to use geolocation
+	in order to disallow access to the site from outside the US.
 -->
-<ruleset name="OptOutPrescreen.com" default_off='failed ruleset test'>
+<ruleset name="OptOutPrescreen.com">
 	<target host="optoutprescreen.com" />
 	<target host="www.optoutprescreen.com" />
 
 	<securecookie host="^www\.optoutprescreen\.com$"
 			name=".+" />
 
+	<test url="http://optoutprescreen.com/" />
+	<test url="http://www.optoutprescreen.com/" />
+	<test url="https://optoutprescreen.com/" />
+
 	<rule from="^(?:http://(?:www\.)?|https://)optoutprescreen\.com/"
```

Comment:
While you're in this file, could you update this rule to match the style I described in https://github.com/EFForg/https-everywhere/pull/2538/files?